### PR TITLE
pd: use pingcap/log.ZapEncoding

### PR DESCRIPTION
**Pull request by 9547**
implement step 2 as discussed in https://github.com/tikv/pd/pull/3480#issuecomment-799961956

### CODE REVIEW

**Comment by 9547** [comment]
/cc @HunDunDM 

PTAL

File: pkg/pd/etcd.go [hunk 1]
```diff
@@ -21,31 +21,14 @@ import (
...
-}
-
 func NewEtcdClient(lc fx.Lifecycle, config *config.Config) (*clientv3.Client, error) {
-	// Because etcd client does not support setting logger directly,
-	// the configuration of pingcap/log is copied here.
 	zapCfg := zap.NewProductionConfig()
-	zapCfg.Encoding = "etcd-client"
-	zapCfg.OutputPaths = []string{"stderr"}
-	zapCfg.ErrorOutputPaths = []string{"stderr"}
+	zapCfg.Encoding = log.ZapEncodingName
```
[the remark this answers is not in the document]
Comment (9547) [reply pkg/pd/etcd.go:31]: Sorry, forgot it.

**Review by HunDunDM — APPROVED** [review]

**Comment by HunDunDM** [comment]
@breeswish PTAL~
After merging, we need a release to update PD.

**Pull request merged**